### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shanamatthews**
don't point to old github repo of modules anymore, point to MS Learn

### CODE REVIEW

**Comment by shanamatthews** [comment]
@GeekTrainer

File: README.md [hunk 1]
```diff
@@ -27,8 +27,10 @@ You can get started by creating an account on Azure. There's three different opt
 
 ## Student Resources
 
-- [Azure Student Labs](https://aka.ms/azurestudentlabs) - Our repository of labs geared for computer science students that you can learn from. Find them at [aka.ms/azurestudentlabs](https://aka.ms/azurestudentlabs)
+- [Learning paths for students](https://aka.ms/student-learn) - Hands-on learning paths and modules designed for students. Find them on Microsoft Learn at [aka.ms/student-learn](https://aka.ms/student-learn)
```
Comment (GeekTrainer) [inline README.md]: Can we remove the callout to aka.ms itself as it's not accessible? Also, a couple of sentences on what the learn content looks like/is?

Thanks!

Comment (shanamatthews) [reply README.md]:
```suggestion
[Learning paths for students](https://aka.ms/StudentsLearn) - Hands-on learning paths and modules designed for students. Find them on Microsoft Learn at [Learning paths for students - aka.ms/StudentsLearn](https://aka.ms/StudentsLearn)

```

File: README.md [hunk 2]
```diff
@@ -27,8 +27,11 @@ You can get started by creating an account on Azure. There's three different opt
 
 ## Student Resources
 
-- [Azure Student Labs](https://aka.ms/azurestudentlabs) - Our repository of labs geared for computer science students that you can learn from. Find them at [aka.ms/azurestudentlabs](https://aka.ms/azurestudentlabs)
+[Learning paths for students](https://aka.ms/StudentsLearn) - Hands-on learning paths and modules designed for students. Find them on Microsoft Learn at [Learning paths for students - aka.ms/StudentsLearn](https://aka.ms/StudentsLearn)
+
```
Comment (shanamatthews) [inline README.md]:
```suggestion

```

File: README.md [hunk 3]
```diff
@@ -27,8 +27,10 @@ You can get started by creating an account on Azure. There's three different opt
 
 ## Student Resources
 
-- [Azure Student Labs](https://aka.ms/azurestudentlabs) - Our repository of labs geared for computer science students that you can learn from. Find them at [aka.ms/azurestudentlabs](https://aka.ms/azurestudentlabs)
+[Learning paths for students](https://aka.ms/StudentsLearn) - Hands-on learning paths and modules designed for students. Find them on Microsoft Learn at [Learning paths for students - aka.ms/StudentsLearn](https://aka.ms/StudentsLearn)
```
Comment (shanamatthews) [inline README.md]:
```suggestion
- [Learning paths for students](https://aka.ms/StudentsLearn) - Hands-on learning paths and modules designed for students. Learn about the cloud, AI, and even how to code for free with these short, hands-on modules. Find them on Microsoft Learn at [Learning paths for students - aka.ms/StudentsLearn](https://aka.ms/StudentsLearn)
```

**Comment by GeekTrainer** [comment]
Best PR Ever!

**Pull request merged**